x-pack/test/observability_functional/apps/observability/alerts/pagination.ts

```diff
@@ -0,0 +1,136 @@
+      });
+
+      it('Does not render pagination controls', async () => {
+        await observability.alerts.pagination.missingPrevPageButtonOrFail();
+      });
+    });
+
+    describe(`When more than ${ROWS_COUNT_TO_HIDE_PAGE_SELECTOR} alerts are visible in the screen`, () => {
+      before(async () => {
+        // current archiver has 12 open alerts
+        await observability.alerts.common.setWorkflowStatusFilter('open');
+      });
+
+      describe('Page size selector', () => {
+        it('Renders page size selector', async () => {
+          await observability.alerts.pagination.getPageSizeSelectorOrFail();
+        });
+
+        it('Default rows per page selector is 50', async () => {
+          await retry.try(async () => {
+            const defaultAlertsPerPage = await (
+              await observability.alerts.pagination.getPageSizeSelector()
+            ).getVisibleText();
+            expect(defaultAlertsPerPage).to.contain(DEFAULT_ROWS_PER_PAGE);
+          });
+        });
+
+        it('10 rows per page selector works', async () => {
+          await retry.try(async () => {
+            await (await observability.alerts.pagination.getPageSizeSelector()).click();
+            await (await observability.alerts.pagination.getTenRowsPageSelector()).click();
+            const tableRows = await observability.alerts.common.getTableCellsInRows();
+            expect(tableRows.length).to.not.be.greaterThan(10);
+          });
+        });
+
+        it('25 rows per page selector works', async () => {
+          await retry.try(async () => {
+            await (await observability.alerts.pagination.getPageSizeSelector()).click();
+            await (await observability.alerts.pagination.getTwentyFiveRowsPageSelector()).click();
+            const tableRows = await observability.alerts.common.getTableCellsInRows();
+            expect(tableRows.length).to.not.be.greaterThan(25);
+          });
+        });
+      });
+
+      describe('Pagination controls', () => {
+        before(async () => {
+          await (await observability.alerts.pagination.getPageSizeSelector()).click();
+          await (await observability.alerts.pagination.getTenRowsPageSelector()).click();
+        });
+        beforeEach(async () => {
+          await observability.alerts.pagination.goToFirstPage();
+        });
+
+        it('Renders previous page button', async () => {
+          await observability.alerts.pagination.getPrevPageButtonOrFail();
+        });
+
+        it('Renders next page button', async () => {
+          await observability.alerts.pagination.getNextPageButtonOrFail();
+        });
+
+        it('Previous page button is disabled', async () => {
+          const prevButtonDisabledValue =
+            await observability.alerts.pagination.getPrevButtonDisabledValue();
+          expect(prevButtonDisabledValue).to.be('true');
+        });
+
+        it('Next page button works', async () => {
```

Comment:
Nit: I have a pet peeve for tests that include the words "works" and "correct" :D 
If I could suggest to re-word this (and the one below) as "it('is possible to go to the next page')"?